app/controllers/collaborators_controller.rb

```diff
@@ -5,8 +5,7 @@ class CollaboratorsController < ApplicationController
 
   def show
     authorize @collaborator
-
-    render :layout => "profiles"
+    @container_fluid = true
```

Comment:
Esta no es la solución más limpia. Te recomiendo para este tipo de situaciones utilizar los helpers de rails. Una solución más adecuada es crear un helper y llamarlo en la vista. 

Lo que quieres hacer es en casos específicos (el controlador te dice que acciones por lo general) hacer un render de otro tipo de container. @zoek1 

http://www.rails-dev.com/custom-view-helpers-in-rails-4
http://stackoverflow.com/questions/3757491/can-i-get-the-name-of-the-current-controller-in-the-view